Extension/src/Debugger/configurationProvider.ts

```diff
@@ -295,6 +311,12 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
             // logger.showOutputChannel();
         }
 
+        // Run deploy steps
+        const deploySucceeded: boolean = await this.deploySteps(config, folder, token);
+        if (!deploySucceeded || token?.isCancellationRequested) {
+            return;
```

Comment:
returning `null` will cause a pop-up notification 

```suggestion
            return undefined;
```

Reply:
`return;` is equivalent to  `return undefined;`. Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return

> Syntax: return [expression];
expression
The expression whose value is to be returned. If omitted, undefined is returned instead.

Reply:
I would still prefer it to be displayed to keep with the other `return`s in this method.

Reply:
ok